### DISCUSSION TITLE
will you be adding adblockers for the browser version

### DISCUSSION THREAD

**Discussion by JunkyardCat** [arianneorpilla/jidoujisho#246] [General]
So i noticed you incorporated a browser in nidou and was wondering if in the future you plan on adding adblockers?

**Comment by arianneorpilla** [comment]
It's possible, just a matter of collating a list of hosts. I would prefer to crowdsource for that instead of collate that myself as I'm quite busy at the moment. In the meantime, I recommend an OS-wide or VPN-based adblock.

**Comment by JunkyardCat** [comment]
Theres probably some list free right now like the list from pi-hole and adguard or ublock

**Comment by arianneorpilla** [comment]
It might be better if this list is user provided instead rather than me dictating the block. As much as I also dislike ads, what list is there should be opt in rather than opt out, and customisable.
